jina/serve/runtimes/gateway/http/app.py

```diff
@@ -140,7 +143,7 @@ async def _status():
             tags=['Debug']
             # do not add response_model here, this debug endpoint should not restricts the response model
         )
-        async def post(body: JinaEndpointRequestModel):
+        async def post(body: JinaEndpointRequestModel, response: Response):
```

Comment:
are we sure about this? why is response passed as an input argument?

Reply:
This is not our `Response`, but a response object handled by FastAPI. This is needed when we want to manually set the HTTP status code: https://fastapi.tiangolo.com/advanced/response-change-status-code/#use-a-response-parameter